chemprop/train/make_predictions.py

```diff
@@ -259,6 +272,8 @@ def make_predictions(args: PredictArgs, smiles: List[List[str]] = None,
                  loading data and a model and making predictions.
     :param smiles: List of list of SMILES to make predictions on.
     :param model_objects: Tuple of output of load_model function which can be called separately.
+    :param return_invalid_smiles: Whether to return None values for invalid SMILES, otherwise will skip them in returned predictions.
```

Comment:
Would it make sense to make the docstring description of `return_invalid_smiles` identical in both `predict_and_save()`  and `make_predictions()`, or is it different on purpose?

Reply:
This was unintentional, I will make them the same.